use_cases/Land_Use_Land_Cover/METplus/GridStat_apcp_lulc.conf

```diff
@@ -0,0 +1,190 @@
+# Time Info
+# LOOP_BY options are INIT, VALID, RETRO, and REALTIME
+# If set to INIT or RETRO:
+#   INIT_TIME_FMT, INIT_BEG, INIT_END, and INIT_INCREMENT must also be set
+# If set to VALID or REALTIME:
+#   VALID_TIME_FMT, VALID_BEG, VALID_END, and VALID_INCREMENT must also be set
+# LEAD_SEQ is the list of forecast leads to process
+# https://metplus.readthedocs.io/en/latest/Users_Guide/systemconfiguration.html#timing-control
+###
+
+LOOP_BY = VALID
+VALID_TIME_FMT = %Y%m%d%H%M
+VALID_BEG = 201707031300
+VALID_END = 201707050000
+VALID_INCREMENT = 3600
+
+LEAD_SEQ = 1H
+
+
+###
+# File I/O
+# https://metplus.readthedocs.io/en/latest/Users_Guide/systemconfiguration.html#directory-and-filename-template-info
+###
+
+FCST_PCP_COMBINE_INPUT_DIR = /data/input/wrf
+FCST_PCP_COMBINE_INPUT_TEMPLATE = wrfout_d03_{valid?fmt=%Y-%m-%d_%H%M}.morr300.dfw4x.nc
+
+FCST_GRID_STAT_INPUT_DIR = {OUTPUT_BASE}/pcp_combine/wrf/APCP
+FCST_GRID_STAT_INPUT_TEMPLATE = wrfout_d03_{valid?fmt=%Y%m%d%H%M}.morr300.dfw4x.apcp.A{level?fmt=%HH}h.nc
+
+OBS_GRID_STAT_INPUT_DIR = {OUTPUT_BASE}/pcp_combine/obs
+OBS_GRID_STAT_INPUT_TEMPLATE = RADAR_DFW22_N1P_d02_{valid?fmt=%Y%m%d%H%M}_A{level?fmt=%HH}h.nc
+
+GRID_STAT_OUTPUT_DIR = {OUTPUT_BASE}/grid_stat
+GRID_STAT_OUTPUT_TEMPLATE = apcp
+
+LOG_LEVEL = DEBUG
+
+
+###
+# Field Info
+# https://metplus.readthedocs.io/en/latest/Users_Guide/systemconfiguration.html#field-info
+###
+
+MODEL = WRF
+OBTYPE = NEXRAD
+
+BOTH_VAR1_NAME = APCP
+BOTH_VAR1_LEVELS = A01
+BOTH_VAR1_THRESH = gt0.0,ge0.254,ge0.508,ge1.27,ge2.54,ge3.810,ge6.350,ge8.890,ge12.700,ge25.400
+
+
+###
+# PCPCombine Settings
+# https://metplus.readthedocs.io/en/latest/Users_Guide/wrappers.html#pcpcombine
+###
+
+PCP_COMBINE_SKIP_IF_OUTPUT_EXISTS = False
+FCST_PCP_COMBINE_RUN = True
+
+FCST_PCP_COMBINE_MAX_FORECAST = 1d
+FCST_PCP_COMBINE_CONSTANT_INIT = FALSE
+FCST_PCP_COMBINE_INPUT_DATATYPE = NETCDF
+
+
+###
+# GridStat Settings
+# https://metplus.readthedocs.io/en/latest/Users_Guide/wrappers.html#gridstat
+###
+
+LOG_GRID_STAT_VERBOSITY = 4
+
+GRID_STAT_REGRID_TO_GRID =  FCST
+GRID_STAT_REGRID_VLD_THRESH = 0.5
+GRID_STAT_REGRID_METHOD = BUDGET
+GRID_STAT_REGRID_WIDTH = 2
+GRID_STAT_REGRID_SHAPE = SQUARE
+
+GRID_STAT_NEIGHBORHOOD_FIELD = BOTH
+GRID_STAT_NEIGHBORHOOD_WIDTH = 3,5,7
+GRID_STAT_NEIGHBORHOOD_SHAPE = SQUARE
+GRID_STAT_NEIGHBORHOOD_COV_THRESH = >=0.5
+
+GRID_STAT_OUTPUT_FLAG_FHO = STAT
+GRID_STAT_OUTPUT_FLAG_CTC = STAT
+GRID_STAT_OUTPUT_FLAG_CTS = STAT
+GRID_STAT_OUTPUT_FLAG_CNT = STAT
+GRID_STAT_OUTPUT_FLAG_SL1L2 = STAT
+GRID_STAT_OUTPUT_FLAG_NBRCTC = STAT
+GRID_STAT_OUTPUT_FLAG_NBRCTS = STAT
+GRID_STAT_OUTPUT_FLAG_NBRCNT = STAT
+
+GRID_STAT_NC_PAIRS_FLAG_LATLON = FALSE
+GRID_STAT_NC_PAIRS_FLAG_RAW = FALSE
+GRID_STAT_NC_PAIRS_FLAG_DIFF = FALSE
+GRID_STAT_NC_PAIRS_FLAG_CLIMO = FALSE
+GRID_STAT_NC_PAIRS_FLAG_WEIGHT = FALSE
+GRID_STAT_NC_PAIRS_FLAG_NBRHD = FALSE
+GRID_STAT_NC_PAIRS_FLAG_APPLY_MASK = FALSE
+
+
+[dir]
+OUTPUT_BASE = /glade/work/briannen/iwrf/output
```

Comment:
I think this line and the [dir] line above it should be removed. Same in the other use case conf file.